desktop/core/base_requirements.txt

```diff
@@ -80,7 +79,5 @@ drf-spectacular[sidecar]==0.27.2
 -e file://${ROOT}/desktop/core/ext-py3/boto-2.49.0
 -e file://${ROOT}/desktop/core/ext-py3/django-axes-5.13.0
 -e file://${ROOT}/desktop/core/ext-py3/djangosaml2-0.18.0
--e file://${ROOT}/desktop/core/ext-py3/dnspython-1.16.0  # Required for eventlet==0.30.2 and python3.10
--e file://${ROOT}/desktop/core/ext-py3/eventlet-0.30.2
```

Comment:
Can you also remove these packages from `/ext-py3` directory?

Reply:
done.